packages/core/src/markdown-summary.ts

```diff
@@ -0,0 +1,384 @@
+import {EOL} from 'os'
+import {constants, promises} from 'fs'
+const {access, appendFile, stat, writeFile} = promises
+
+export const SUMMARY_LIMIT_BYTES = 128_000
```

Comment:
Just curious if we want some sort of explanation here of the limit? Maybe down the road we point to the docs

Reply:
That's a good point. I'll add the docs as a comment and in the error message

Reply:
Added in https://github.com/actions/toolkit/pull/1014/commits/6295f5d25bb8417c8705fd29b31ce4f3bb743302